src/Product/ProductList.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from "react";
+import Product from "./Product";
+
+const API_URL = "https://px32id5fdg.execute-api.us-east-1.amazonaws.com/data";
+
+function ProductList() {
+  const [productData, setProductData] = useState([]);
+  const [searchInput, setSearchInput] = useState("");
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    fetch(API_URL)
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        setProductData(data);
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }, []);
+
+  const filteredProductData = productData
+    .filter((product) => {
+      return (
+        product.name.toLowerCase().includes(searchInput.toLocaleLowerCase()) ||
+        product.shortDescription
+          .toLowerCase()
+          .includes(searchInput.toLocaleLowerCase())
```

Comment:
Just curious (I should know this but I don't), how does `toLocaleLowerCase()` differ from `toLowerCase()`?

Another thing to consider: `searchInput.toLocaleLowerCase()` is called twice in the same or statement. It doesn't really matter for this app, but technically it is inefficient to do this twice in a row. Instead, you could call `toLocaleLowerCase()` one time before the or statement, and save the result to a variable.